docs/src/examples/components/Menu/commonSteps.ts

```diff
@@ -0,0 +1,30 @@
+import { Menu } from '@stardust-ui/react'
+
+interface StepsOptions {
+  vertical?: boolean
+}
+
+const menuSelector = `.${Menu.className}`
+const selectors = {
+  menu: menuSelector,
+  item: (itemIndex: number) => `${menuSelector} li:nth-child(${itemIndex}) a`,
+}
+
+const commonSteps = [
+  steps => steps.hover(selectors.item(2)).snapshot('Hovers 2nd item (hover state styles)'),
+  steps => steps.click(selectors.item(2)).snapshot('Clicks on 2nd item (active state styles)'),
+]
+
+const getSteps = ({ vertical }: StepsOptions = {}) => [
+  ...commonSteps,
+  (steps, keys) =>
+    steps
+      .keys(selectors.item(2), vertical ? keys.downArrow : keys.rightArrow)
+      .snapshot('Navigates to next item (focus state styles)'),
+  (steps, keys) =>
+    steps
+      .keys(selectors.item(3), vertical ? keys.upArrow : keys.leftArrow)
+      .snapshot('Navigates to previous item (active and focus state styles)'),
+]
+
+export default getSteps
```

Comment:
why didn't the linter complain about the lack of end empty line?

Reply:
github doesn't show it I think, locally it's there..